handlers.go

```diff
@@ -12,6 +12,7 @@ import (
 func StatusHandler(w http.ResponseWriter, r *http.Request) {
 	body, _ := ioutil.ReadAll(r.Body)
 
+	// TODO: make use of request struct from models.go
```

Comment:
Yes, why not use the model types? Is it more complicated?

Reply:
I just added the models relatively recently, will do.